webpack/JobInvocationDetail/JobInvocationConstants.js

```diff
@@ -29,3 +42,78 @@ export const DATE_OPTIONS = {
   hour12: false,
   timeZoneName: 'short',
 };
+
+const Columns = () => {
+  const getColumnsStatus = ({ hostJobStatus }) => {
+    switch (hostJobStatus) {
+      case 'success':
+        return { title: __('Succeeded'), status: 0 };
+      case 'error':
+        return { title: __('Failed'), status: 1 };
+      case 'planned':
+        return { title: __('Scheduled'), status: 2 };
+      case 'running':
+        return { title: __('Pending'), status: 3 };
+      case 'cancelled':
+        return { title: __('Cancelled'), status: 4 };
+      case 'Awaiting start':
+        return { title: __('Awaiting start'), status: 5 };
+      default:
+        return { title: hostJobStatus, status: 6 };
+    }
+  };
+  const hostDetailsPageUrl = useForemanHostDetailsPageUrl();
+
+  return {
+    name: {
+      title: __('Name'),
+      wrapper: ({ name }) => (
+        <a href={`${hostDetailsPageUrl}${name}`}>{name}</a>
+      ),
+      weight: 1,
+    },
+    groups: {
+      title: __('Host group'),
+      wrapper: ({ hostgroup_id, hostgroup_name }) => (
+        <a href={`/hostgroups/${hostgroup_id}/edit`}>{hostgroup_name}</a>
+      ),
+      weight: 2,
+    },
+    os: {
+      title: __('OS'),
+      wrapper: ({ operatingsystem_id, operatingsystem_name }) => (
+        <a href={`/operatingsystems/${operatingsystem_id}/edit`}>
+          {operatingsystem_name}
+        </a>
+      ),
+      weight: 3,
+    },
+    smart_proxy: {
+      title: __('Smart proxy'),
+      wrapper: ({ smart_proxy_name, smart_proxy_id }) => (
+        <a href={`/smart_proxies/${smart_proxy_id}`}>{smart_proxy_name}</a>
+      ),
+      weight: 4,
+    },
+    status: {
+      title: __('Status'),
+      tableTitle: ({ job_status }) =>
+        getColumnsStatus({ hostJobStatus: job_status }).title,
+      status: ({ job_status }) =>
+        getColumnsStatus({ hostJobStatus: job_status }).status,
+      wrapper: ({ job_status }) => {
+        const { title, status } = getColumnsStatus({
```

Comment:
Its still 3 calls instead of one, I think that it can call `getColumnsStatus` only once